Simulator.py

```diff
@@ -0,0 +1,492 @@
+def print_basic_parameters():
+	"""
+	This methods prints the basic parameters of the epidemiological model.
+	"""
+
+def print_populations():
+	"""
+	This methods prints parameters of the population model.
+	"""
+
+def print_immunity_model():
+	"""
+	This methods prints the basic parameters of the immunity model.
+	"""
+
+def print_all(basic_parameters=True, populations=True, immunity_model=True):
+	"""
```

Comment:
Mixing tabs and spaces is a recipe for pain down the road. Best reindent to 4 spaces everywhere